### ISSUE TITLE
Publish releases and Docker images for arm64

### ISSUE THREAD

**Issue by errorhandler** [moov-io/ach-test-harness#130]
Currently releases and docker images are only for amd64. Building the Docker image manually on arm64 works perfectly fine but it would be great if the official releases build for arm64.

**Comment by adamdecaf** [comment]
Agreed. See https://github.com/moov-io/achgateway/issues/159#issuecomment-1351950261

**Comment by adamdecaf** [comment]
https://github.com/actions/runner-images/issues/9255
